website/blog/2025-04-14-release-1.18.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Podman Desktop 1.18 Release
+description: Podman Desktop 1.18 has been released!
+slug: podman-desktop-release-1.18
+authors: SoniaSandler
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.18/banner.png
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.18 Release! 🎉
+
+![podman-desktop-hero-1.18](/img/blog/podman-desktop-release-1.18/banner.png)
+
+Podman Desktop 1.18 is now available! [Click here to download it](/downloads)!
+
+This release brings exciting new features and improvements:
+
+- **Refining the Docker Compatibility feature**: The Docker Compatibility page has been moved out of experimental mode, and now it helps you fully set up Podman even when installed through `brew`. It also detects and notifies you when Podman can't bind the Docker socket.
+- **Switching Kubernetes namespace**: Switching between Kubernetes namespaces within Podman Desktop is now available
+- **Jobs in Kubernetes**: You can now see Kubernetes jobs in our growing Kubernetes section of Podman Desktop
+- **Experimental status bar providers enhancements**: Status bar providers can be pinned and unpinned based on user preferences and the provider status icons have been updated to be easier to understand
```

Comment:
first entry ends with a period, not the others. I'm not sure sure which pattern to follow, but it should be consistent